tests/providers/google/suite/operators/test_sheets.py

```diff
@@ -39,12 +39,9 @@ def test_execute(self, mock_xcom, mock_hook):
         op = GoogleSheetsCreateSpreadsheetOperator(
             task_id="test_task", spreadsheet=spreadsheet, gcp_conn_id=GCP_CONN_ID
         )
-        op.execute(context)
+        op_execute_result = op.execute(context)
 
         mock_hook.return_value.create_spreadsheet.assert_called_once_with(spreadsheet=spreadsheet)
 
-        calls = [
-            mock.call(context, "spreadsheet_id", SPREADSHEET_ID),
-            mock.call(context, "spreadsheet_url", SPREADSHEET_URL),
-        ]
-        mock_xcom.has_calls(calls)
+        assert op_execute_result['spreadsheetId'] == '1234567890'
+        assert op_execute_result['spreadsheetUrl'] == 'https://example/sheets'
```

Comment:
Does `op_execute_result` contain other values? We should try to verify the _entire_ dict if possible.

Reply:
Thank you, I will write the entire dict's results and write more tests wherever appropriate.

Reply:
I just printed the dict entirely and saw that these are the only two values the `GoogleSheetsCreateSpreadsheetOperator`'s execute call returns. 
The dict looks like this: `{'spreadsheetId': '1234567890', 'spreadsheetUrl': 'https://example/sheets'}`.